test/dynamo/test_modules.py

```diff
@@ -1212,6 +1213,7 @@ def test_torch_static():
         )
         self.assertEqual(cnt.frame_count, 1, "No guards should have triggered.")
 
+    @patch.object(torch._dynamo.config, "dynamic_shapes", False)
```

Comment:
It would be good to know if these *have* to be disabled for dynamic shapes, or if in principle they shouldn't be and it's a matter of hosing down bugs.

Reply:
This is actually just code motion

They used to be in:

```
unittest.expectedFailure(
    DynamicShapesNNModuleTests.test_lazy_module2_dynamic_shapes
    # RuntimeError: SymIntArrayRef expected to contain only concrete integers
)
```